l10n_th_withholding_tax_report/report/report_withholding_tax_csv.py

```diff
@@ -14,53 +13,6 @@
     _logger.debug("Can not import csvwriter`.")
 
 
-class WithholdingTaxCSV(models.AbstractModel):
```

Comment:
Can you explain whey this is being removed?

Reply:
Because this models is not used. I don't know why to create this models?